tests/framework/pkg/schemas/cluster.x-k8s.io.machines/v1beta1/schema.go

```diff
@@ -0,0 +1,27 @@
+package schema
+
+import (
+	"github.com/rancher/norman/types"
+	provisioningSchema "github.com/rancher/rancher/tests/framework/pkg/schemas/provisioning.cattle.io/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	capi "sigs.k8s.io/cluster-api/api/v1beta1"
+)
+
+var (
+	Version = types.APIVersion{
+		Version: capi.GroupVersion.Version,
+		Group:   capi.GroupVersion.Group,
+		Path:    "/v1",
+	}
+
+	Schemas = ClusterMachineSchemas(&Version).
+		Init(clusterMachineTypes)
+)
+
+func clusterMachineTypes(schemas *types.Schemas) *types.Schemas {
```

Comment:
I saw there were comments added for the non-generated functions, should there be one here as well?

Reply:
I didn't since this is a non-exported function (ie private) so it wouldn't show up in our godocs any way.